tests/unit/ClientTest.php

```diff
@@ -46,4 +46,20 @@ public function testClientFeed()
         $client = new Client('key', 'secret', $location='qa');
         $feed1 = $client->feed('flat', '1');
     }
+
+    public function testEnvironmentVariable()
+    {
+        // Arrange
+        putenv('STREAM_BASE_URL=test.stream-api.com/api');
+        $client = new Client('key', 'secret');
+
+        // Act
+        $baseUrl = $client->getBaseUrl();
+
+        // Assert
+        $this->assertSame('test.stream-api.com/api', $baseUrl);
+
+        // Teardown
+        putenv('STREAM_BASE_URL=');
```

Comment:
cleanup is not 100% correct, you assume previous value of STREAM_BASE_URL is '' which is not certain

Reply:
You're right. I searched and found that doing `putenv('STREAM_BASE_URL')` does unset the environment variable. Let's do that.